pkg/blob/blob.go

```diff
@@ -693,6 +694,37 @@ func getStorageAccount(secrets map[string]string) (string, string, error) {
 	return accountName, accountKey, nil
 }
 
+// get account sas token if existed from secrets map
+func getAccountSasToken(secrets map[string]string, accountName string) (string, error) {
+	if secrets == nil {
```

Comment:
I think it's ok secrets is nil

Reply:
In this func I want to get sas token from secret if existed, if it returns error I will generate sas instead.

Reply:
if secrets is nil, then it would not go into L704, so finally you wll get an error

Reply:
yes, I delete it.